tests/models/test_measurements.py

```diff
@@ -171,3 +171,25 @@ def test_measurements_cannot_be_saved_twice(ptf_datastore):
                 session.commit()
 
 
+def test_measurements_forced_photometry(ptf_datastore):
+    offset_max = 2.0
+    for m in ptf_datastore.measurements:
+        if abs(m.offset_x) < offset_max and abs(m.offset_y) < offset_max:
+            break
+    else:
+        raise RuntimeError(f'Cannot find any measurement with offsets less than {offset_max}')
+
+    flux_small_aperture = m.get_flux_at_point(m.ra, m.dec, aperture=1)
+    flux_large_aperture = m.get_flux_at_point(m.ra, m.dec, aperture=len(m.aper_radii) - 1)
+    flux_psf = m.get_flux_at_point(m.ra, m.dec, aperture=-1)
+    assert flux_small_aperture[0] == pytest.approx(m.flux_apertures[1], abs=0.01)
```

Comment:
These verify that a new run of `get_flux_at_point` returns what is expected.

However, looking at the fixtures, it looks like the values inside `ptf_datastore` are created using a `Measurer`, which (I believe-- I haven't dug through the code to verify) will have itself been done through a `get_flux_at_point` call.

Are there any explicit tests of `get_flux_at_point` itself?  (That is, make sure it's doing what we expect, not that it does the same thing every time.)

Reply:
The PSF flux is calculated using the same code, but the aperture flux is gotten through the `photometry` module. 
I have some tests for that code in `test_measuring` and it seems to be doing the right thing, but maybe I can add something to test the PSF photometry over there.